cypress/integration/login_page.spec.js

```diff
@@ -21,32 +23,74 @@ describe("Login Page", () => {
       cy.get("[data-testid='alert']").should("be.visible");
     });
 
-    it("Displays an error message when submitting a form and getting an unsuccessful reply from the server.", () => {
-      cy.intercept("POST", "/api/token/temporary", {
-        statusCode: 403,
-      }).as("userSuccess");
-      cy.get("input[id='loginEmail']").type("test@cds-snc.ca");
+    it("Displays an error message when submitting an empty username.", () => {
       cy.get("[type='submit']").click();
-      cy.get("[data-testid='alert']").should("be.visible");
+      cy.get("[id='errorMessageusername']").should(
```

Comment:
The login validation tests could also go outside of the Bearer Token State tests

Reply:
I will remove these tests as they no longer apply